.github/ISSUE_TEMPLATE/bug-report.yml

```diff
@@ -0,0 +1,54 @@
+name: Bug report
+description: anndata doesn’t do what it should? Please help us fix it!
+#title: ...
+labels: ['Bug 🐛']
+#assignees: []
+body:
+- type: checkboxes
+  id: terms
+  attributes:
+    label: Please make sure these conditions are met
+    # description: ...
+    options:
+    - label: I have checked that this issue has not already been reported.
+      required: true
+    - label: I have confirmed this bug exists on the latest version of anndata.
+      required: true
+    - label: (optional) I have confirmed this bug exists on the master branch of anndata.
+      required: false
+- type: markdown
+  attributes:
+    value: |
+      **Note**: Please read [this guide](https://matthewrocklin.com/blog/work/2018/02/28/minimal-bug-reports) detailing how to provide the necessary information for us to reproduce your bug.
+- type: textarea
+  id: description
+  attributes:
+    label: What happened?
+    description: Describe what you tried to do, and what happened instead.
+  validations:
+    required: true
+- type: textarea
+  id: code
+  attributes:
+    label: Minimal code sample
+    description: (that we can copy&paste without having any data)
+    render: python
+  validations:
+    required: true
+- type: textarea
+  id: traceback
+  attributes:
+    label: Error output
+    description: Produced by the code above. Leave empty if the issue is unexpected behavior instead of an error.
+    render: pytb
+  validations:
+    required: false
```

Comment:
```suggestion
- type: textarea
  id: Report
  attributes:
    label: Report
    description: Describe the bug you encountered, and what you were trying to do. Please use [github markdown](https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax) features for readability.
  validations:
    required: true
```

Reply:
In my experience, that leads to too unspecific reports. I’d rather have people write “nothing” in the code field in the rare case that it’s not about code but installation or so than have people forget to paste a stacktrace.